packages/vercel/src/index.ts

```diff
@@ -468,6 +474,23 @@ export default function vercelAdapter({
 					}
 				}
 
+				let trailingSlash: boolean | undefined;
+
+				if (_config.trailingSlash && _config.trailingSlash !== 'ignore') {
+					trailingSlash = _config.trailingSlash === 'always';
+				}
+
+				const { routes: redirects = [], error } = getTransformedRoutes({
+					trailingSlash,
+					rewrites: [],
+					redirects: getRedirects(routes, _config),
+					headers: [],
+				});
+
+				if (error) {
+					logger.error(`Error generating redirects: ${error.message}`);
```

Comment:
Is there a particular reason why we don't throw an error instead? Might as well add a comment just in case